easybuild/easyconfigs/b/BEDTools/BEDTools-2.31.1-GCC-13.3.0.eb

```diff
@@ -0,0 +1,53 @@
+# Author: Maxime Schmitt, University of Luxembourg
+# Author: Adam Huffman, The Francis Crick Institute
+#
+# Based on the work of: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics (SIB)
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = 'BEDTools'
+version = '2.31.1'
+
+homepage = 'https://bedtools.readthedocs.io/'
+description = """BEDTools: a powerful toolset for genome arithmetic.
+The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps and
+computing coverage.
+The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF, and SAM/BAM."""
+
+toolchain = {'name': 'GCC', 'version': '13.3.0'}
+
+source_urls = ['https://github.com/arq5x/bedtools2/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['79a1ba318d309f4e74bfa74258b73ef578dccb1045e270998d7fe9da9f43a50e']
+
+builddependencies = [
+    ('Python', '3.12.3'),
+]
+dependencies = [
+    ('XZ', '5.4.5'),
+    ('zlib', '1.3.1'),
+    ('bzip2', '1.0.8'),
+    ('BamTools', '2.5.2'),
+]
+
+buildopts = 'CXX="$CXX"'
+
+files_to_copy = [
+    'bin',
+    'docs',
+    'data',
+    'genomes',
+    'scripts',
+    'test',
+]
+
+sanity_check_paths = {
+    'files': ['bin/%(namelower)s', 'bin/pairToBed', 'bin/mergeBed', 'bin/bedToBam', 'bin/fastaFromBed'],
+    'dirs': ['bin', 'docs', 'data', 'genomes', 'scripts', 'test'],
```

Comment:
Same here, I preferred the use of `files_to_copy`.

Reply:
done